tests/e2e/suite_test.go

```diff
@@ -32,6 +32,7 @@ import (
 const (
 	kcpConfigEnvVar, skrConfigEnvVar = "KCP_KUBECONFIG", "SKR_KUBECONFIG"
 	clientQPS, clientBurst           = 1000, 2000
+	istioNamespace                   = "istio-system"
```

Comment:
actually I have a better place to declaire namespace used for tests, https://github.com/kyma-project/lifecycle-manager/pull/2030/files#diff-5dab9c6f73d5f7fde3503baa47c3f8ef0528addf1143961dac081556181d7ebbR15-R16, if you PR is merged after that one, please update accordingly, otherwise, do this in another PR.